src/main/kotlin/app/cash/licensee/task.kt

```diff
@@ -218,8 +225,7 @@ abstract class LicenseeTask : DefaultTask() {
     val artifactsJson = outputFormat.encodeToString(listOfArtifactDetail, artifactDetails)
 
     val outputDir = outputDir.asFile.get()
-    val artifactsJsonFile = File(outputDir, "artifacts.json")
-    artifactsJsonFile.parentFile.mkdirs()
+    val artifactsJsonFile = File(outputDir, ARTIFACTS_JSON)
```

Comment:
Can these consume the providers? That way we don't need to rely on the logic of creating the provider and this file to always stay in sync.

Reply:
Yes, good idea.